smartsim/launcher/pbs/pbsLauncher.py

```diff
@@ -118,6 +118,9 @@ def run(self, step):
         if not step_id and step.managed:
             step_id = self._get_pbs_step_id(step)
         self.step_mapping.add(step.name, step_id, task_id, step.managed)
+
+        time.sleep(5)
```

Comment:
is this still meant to be here?

Reply:
I don't even know why I put it there in the first place.